misk/src/main/kotlin/misk/web/HttpCall.kt

```diff
@@ -11,13 +11,31 @@ import okhttp3.RequestBody
 import okio.BufferedSink
 import okio.BufferedSource
 
+/**
+ * Information about the socket on which a HTTP call arrived.
+ */
+data class SourceSocket (
+  val type: Type = Type.NETWORK,
+  val name: String? = null
```

Comment:
I think this will have a name even for network sockets, so can make it a non-optional `String` without a default value.